responsibleai/responsibleai/_managers/counterfactual_manager.py

```diff
@@ -105,6 +105,11 @@ def _create_diceml_explainer(self, method, continuous_features):
         return dice_explainer
 
     def _add_counterfactual_config(self, new_counterfactual_config):
+        if self._model is None:
+            raise UserConfigValidationException(
+                'Model is required for counterfactual example generation and '
+                'explanations')
```

Comment:
Is this message confusing since we sometimes use "explanations" to mean interpret explanations? Was this meant to cover counterfactual importances?

Reply:
Done. Changed to "feature importances"